docs/2_2_common_mechanisms.adoc

```diff
@@ -671,14 +671,14 @@ The importer calls <<fmi3GetClock>> to inquire the clock activation state.
 |`triggered`
 |[[triggered,`triggered`]]<<fmi3SetClock>> in <<EventMode>>,
 <<fmi3ActivateModelPartition>> in <<ClockActivationMode>>,
-arguments <<clocksAboutToTick>> (<<fmi3DoStep>>) and <<clocksTicked>> (<<fmi3CallbackIntermediateUpdate>>)
+argument <<clocksTicked>> of <<fmi3CallbackIntermediateUpdate>>
```

Comment:
@oliverkotte did you update the c files accordingly?

Reply:
Oliver removed the clocksAboutToTick from doStep - you can see that in the header file.
Are you referring to another change that might be missing still?

Reply:
Nevermind @andreas-junghanns  and @oliverkotte . I've double checked that this is correct! Closing this.